private/bufpkg/bufanalysis/bufanalysis.go

```diff
@@ -103,12 +102,6 @@ type FileInfo interface {
 type FileAnnotation interface {
 	// Stringer returns the string representation in text format.
 	fmt.Stringer
```

Comment:
Should we kill this as well?

Reply:
Stringer is useful (in one test so far) and generic enough (not an encoding format) to be part of the interface. I think we should let this be.

Reply:
It's saying that this is how you do the text format though - we should at least update the comment to not reference the text format and not tie it to the text format

Reply:
Updated the comment